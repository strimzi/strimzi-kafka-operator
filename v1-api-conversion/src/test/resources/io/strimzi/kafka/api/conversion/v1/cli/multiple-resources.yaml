apiVersion: v1
data:
  username: YWRtaW4=
  password: MWYyZDFlMmU2N2Rm
kind: Secret
metadata:
  name: mysecret
type: Opaque
---
apiVersion: kafka.strimzi.io/v1alpha1
kind: KafkaUser
metadata:
  name: my-connect
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
    # Kafka Connects internal topics used to store configuration, offsets or status
    - resource:
        type: group
        name: connect-cluster
      operation: Read
    - resource:
        type: topic
        name: connect-cluster-configs
      operation: Read
    - resource:
        type: topic
        name: connect-cluster-configs
      operation: Describe
    - resource:
        type: topic
        name: connect-cluster-configs
      operation: Write
    - resource:
        type: topic
        name: connect-cluster-configs
      operation: Create
    - resource:
        type: topic
        name: connect-cluster-status
      operation: Read
    - resource:
        type: topic
        name: connect-cluster-status
      operation: Describe
    - resource:
        type: topic
        name: connect-cluster-status
      operation: Write
    - resource:
        type: topic
        name: connect-cluster-status
      operation: Create
    - resource:
        type: topic
        name: connect-cluster-offsets
      operation: Read
    - resource:
        type: topic
        name: connect-cluster-offsets
      operation: Write
    - resource:
        type: topic
        name: connect-cluster-offsets
      operation: Describe
    - resource:
        type: topic
        name: connect-cluster-offsets
      operation: Create
    # Additional topics and groups used by connectors
    # Change to match the topics used by your connectors
    - resource:
        type: group
        name: connect-cluster
      operation: Read
    - resource:
        type: topic
        name: my-topic
      operation: Read
    - resource:
        type: topic
        name: my-topic
      operation: Describe
    - resource:
        type: topic
        name: my-topic
      operation: Write
    - resource:
        type: topic
        name: my-topic
      operation: Create
---
apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaConnect
metadata:
  name: my-connect
#  annotations:
#  # use-connector-resources configures this KafkaConnect
#  # to use KafkaConnector resources to avoid
#  # needing to call the Connect REST API directly
#    strimzi.io/use-connector-resources: "true"
spec:
  version: 2.7.0
  replicas: 1
  bootstrapServers: my-cluster-kafka-bootstrap:9093
  tls:
    trustedCertificates:
      - secretName: my-cluster-cluster-ca-cert
        certificate: ca.crt
  authentication:
    type: tls
    certificateAndKey:
      secretName: my-connect
      certificate: user.crt
      key: user.key
  config:
    group.id: connect-cluster
    offset.storage.topic: connect-cluster-offsets
    config.storage.topic: connect-cluster-configs
    status.storage.topic: connect-cluster-status

---
apiVersion: v1
kind: ConfigMap
metadata:
  name: game-demo
data:
  player_initial_lives: "3"
  ui_properties_file_name: "user-interface.properties"
  game.properties: |
    enemy.types=aliens,monsters
    player.maximum-lives=5
  user-interface.properties: |
    color.good=purple
    color.bad=yellow
    allow.textmode=true