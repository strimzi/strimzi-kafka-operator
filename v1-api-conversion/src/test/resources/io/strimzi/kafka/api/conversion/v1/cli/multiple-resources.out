---
apiVersion: v1
data:
  username: YWRtaW4=
  password: MWYyZDFlMmU2N2Rm
kind: Secret
metadata:
  name: mysecret
type: Opaque
---
apiVersion: kafka.strimzi.io/v1
kind: KafkaUser
metadata:
  name: my-connect
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
    - resource:
        type: group
        name: connect-cluster
      operations:
      - Read
    - resource:
        type: topic
        name: connect-cluster-configs
      operations:
      - Read
    - resource:
        type: topic
        name: connect-cluster-configs
      operations:
      - Describe
    - resource:
        type: topic
        name: connect-cluster-configs
      operations:
      - Write
    - resource:
        type: topic
        name: connect-cluster-configs
      operations:
      - Create
    - resource:
        type: topic
        name: connect-cluster-status
      operations:
      - Read
    - resource:
        type: topic
        name: connect-cluster-status
      operations:
      - Describe
    - resource:
        type: topic
        name: connect-cluster-status
      operations:
      - Write
    - resource:
        type: topic
        name: connect-cluster-status
      operations:
      - Create
    - resource:
        type: topic
        name: connect-cluster-offsets
      operations:
      - Read
    - resource:
        type: topic
        name: connect-cluster-offsets
      operations:
      - Write
    - resource:
        type: topic
        name: connect-cluster-offsets
      operations:
      - Describe
    - resource:
        type: topic
        name: connect-cluster-offsets
      operations:
      - Create
    - resource:
        type: group
        name: connect-cluster
      operations:
      - Read
    - resource:
        type: topic
        name: my-topic
      operations:
      - Read
    - resource:
        type: topic
        name: my-topic
      operations:
      - Describe
    - resource:
        type: topic
        name: my-topic
      operations:
      - Write
    - resource:
        type: topic
        name: my-topic
      operations:
      - Create
---
apiVersion: kafka.strimzi.io/v1
kind: KafkaConnect
metadata:
  name: my-connect
spec:
  version: 2.7.0
  replicas: 1
  bootstrapServers: my-cluster-kafka-bootstrap:9093
  groupId: connect-cluster
  configStorageTopic: connect-cluster-configs
  statusStorageTopic: connect-cluster-status
  offsetStorageTopic: connect-cluster-offsets
  tls:
    trustedCertificates:
    - secretName: my-cluster-cluster-ca-cert
      certificate: ca.crt
  authentication:
    type: tls
    certificateAndKey:
      secretName: my-connect
      certificate: user.crt
      key: user.key
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: game-demo
data:
  player_initial_lives: 3
  ui_properties_file_name: user-interface.properties
  game.properties: |
    enemy.types=aliens,monsters
    player.maximum-lives=5
  user-interface.properties: |-
    color.good=purple
    color.bad=yellow
    allow.textmode=true